docs/src/gravity_wave.md

```diff
@@ -50,4 +50,201 @@ Q(z,c) = \frac{\rho_0}{\rho(z)} \frac{2N(z)B_0(c)}{k[c-u(z)]^3}
+```
+where ``\langle \textbf{T} \rangle = [t_{11}, t_{12}; t_{21}, t_{22}]`` is the tensor that contains orographic information. In the code, we compute the zonal and meridional components separately as 
+```math
+\tau_x = \overline{\rho} \overline{N} (t_{11} \overline{u} + t_{21} \overline{v}),
+```
+```math
+\tau_y = \overline{\rho} \overline{N} (t_{12} \overline{u} + t_{22} \overline{v}).
+```
+
+The base flux is then corrected using Froude number and saturation velocity. Let
+```math
+V_{\tau} = \max(\epsilon_0, - \overline{V} \cdot \frac{\tau}{|\tau|}),
+```
+and given the orographic information ``h_{max}`` and ``h_{min}``, the max and min Froude numbers are computed as
+```math
+Fr_{max} = h_{max} \frac{\overline{N}}{V_{\tau}},
+```
+```math
+Fr_{min} = h_{min} \frac{\overline{N}}{V_{\tau}}.
+```
+We also compute the med Froude number as
+```math
+Fr_{med} = Fr_{crit} + Fr_{int},
+```
+where ``Fr_{crit} = 0.7`` is the critical Froude number for nonlinear flow, ``Fr_{int} = 0.5`` is an arbitrary parameter.
+
+The saturation velocity is computed as
+```math
+U_{sat} = \sqrt{\frac{\overline{\rho}}{\rho_0} \frac{V_{\pmb{\tau}}^3}{\overline{N} L_0}},
+```
+where ``\rho_0 = 1.2`` is the arbitrary density scale, and ``L_0 = 80e3`` is the arbitrary horizontal length scale. 
+
+The following set of intermediate variables (``FrU``'s) are computed to correct the linear base flux:
+```math
+FrU_{sat} = Fr_{crit} * U_{sat},
+```
+```math
+FrU_{min} = Fr_{min} * U_{sat},
+```
+```math
+FrU_{med} = Fr_{med} * U_{sat},
+```
+```math
+FrU_{max} = \max(Fr_{max} * U_{sat}, FrU_{min} + \epsilon_0),
+```
+```math
+FrU_{clp} = \min(FrU_{max}, \max(FrU_{min}, FrU_{sat})),
+```
+```math
+FrU_0 = \frac{U_0}{V_{\tau}}U_{sat},
+```
+and ``U_0=1.0`` is the arbitrary velocity scale.
+
+Now the correct linear drag is computed as 
+```math
+\tau_l = \frac{FrU_{max}^{2+\gamma-\epsilon} - FrU_{min}^{2+\gamma-\epsilon}}{2+\gamma-\epsilon},
+```
+and the propagating and non-propagating parts of the drag are computed as
+```math
+\tau_p = a_0 \left[ \frac{FrU_{clp}^{2+\gamma-\epsilon} - FrU_{min}^{2+\gamma-\epsilon}}{2+\gamma-\epsilon} + FrU_{sat}^{\beta+2} \frac{FrU_{max}^{\gamma-\epsilon-\beta} - FrU_{clp}^{\gamma-\epsilon-\beta}}{\gamma-\epsilon-\beta} \right],
+```
+```math
+\tau_{np} = a_1 \frac{U_{sat}}{1+\beta} \left[ \frac{FrU_{max}^{1+\gamma-\epsilon} - FrU_{clp}^{1+\gamma-\epsilon}}{1+\gamma-\epsilon} - FrU_{sat}^{\beta+1} \frac{FrU_{\max}^{\gamma-\epsilon-\beta} - FrU_{clp}^{\gamma-\epsilon-\beta}}{\gamma-\epsilon-\beta} \right].
+```
+
+Here, $(\gamma, \epsilon, \beta) = (0.4, 0.0, 0.5)$ are arbitrary parameters that describe the mountain shapes.
+
+### Saturation profiles for the propagating component
+The vertical profiles of saturated momentum flux ``\tau_{sat}`` is computed then so that momentum forcing can be obtained for 11d\overline{\arrow{V}}/dt = -\overline{\rho}^{-1}d\tau_{sat}/dz11. This only applies to the propagating part.
```

Comment:
```suggestion
The vertical profiles of saturated momentum flux ``\tau_{sat}`` is computed then so that momentum forcing can be obtained for ``d\overline{\arrow{V}}/dt = -\overline{\rho}^{-1}d\tau_{sat}/dz``. This only applies to the propagating part.
```